app/controllers/hyrax/collections_page_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class Hyrax::CollectionsPageController < ApplicationController
+    include Blacklight::SearchHelper
+    include Blacklight::SearchContext
+    include Blacklight::AccessControls::Catalog
+
+    class_attribute :presenter_class
+    self.presenter_class = Hyrax::CollectionsPagePresenter
+
+    def index
+        @presenter = presenter_class.new(collections)
+    end
+
+    private
+
+    # Return 5 collections
```

Comment:
I'm assuming this comment is leftover from the home page collections controller -- the collections page seems to display as many collections as are visible to the user, not just 5.

Reply:
That was me troubleshooting, you're right. Removed!